benchmark/timings.cpp

```diff
@@ -332,6 +335,42 @@ int main(int argc, const char ** argv)
     forwardKinematics(model,data,qs[_smooth],qdots[_smooth], qddots[_smooth]);
   }
   std::cout << "Second Order Kinematics = \t"; timer.toc(std::cout,NBT);
+
+
+  timer.tic();
+  SMOOTH(NBT)
+  {
+    updateFramePlacements(model,data);
```

Comment:
This test is not relevant, as data keeps to be the same.

Reply:
It runs on the full tree, and takes about 1us on my machine (compared to 3us for the kinematics).
This is significant. I suggest that we keep it please.

Reply:
But, the way you did the benchmark is not correct. You have to update the content of data. Please see the other benchmarks please.

Reply:
I removed it for now.